app/pages/Patient/PatientCard/Form/index.jsx

```diff
@@ -55,6 +58,36 @@ const PatientForm = ({
                   onDataChange()
                 }
               }}
+              error={invalid}
+              helperText={error?.message}
+            />
+          )}
+        />
+        <Controller
+          name="holder"
+          control={control}
+          rules={{ ...VALIDATION_SCHEMA.holder }}
+          render={({ field, fieldState: { invalid, error } }) => (
+            <Autocomplete
+              label="Responsável"
+              startAdornment={<PersonRounded />}
+              options={holders}
+              content={({ cpf }) => (
+                <Typography component="span" color="grey.dark" variant="body2">
+                  {cpf}
+                </Typography>
+              )}
+              InputProps={{
```

Comment:
I'm thinking if `InputProps` gives too much customization than what we want. Instead, we could just pass: `onBlur`, `error` and `helperText` as new props. Also our `TextField` supports `error` and `helperText`. You don't need to pass it as props of `InputProps`